util/chunk/chunk_test.go

```diff
@@ -642,3 +658,209 @@ func BenchmarkChunkMemoryUsage(b *testing.B) {
 		chk.MemoryUsage()
 	}
 }
+
+type seqNumberGenerateExec struct {
+	seq          int
+	genCountSize int
+}
+
+func (x *seqNumberGenerateExec) Next(chk *Chunk, resize bool) {
+	if resize {
+		chk.GrowAndReset(1024)
+	} else {
+		chk.Reset()
+	}
+	for chk.NumRows() < chk.Capacity() {
+		x.seq++
+		if x.seq > x.genCountSize {
+			break
+		}
+		chk.AppendInt64(0, 1)
+	}
+}
+
+func BenchmarkChunkRenewConsumeExec10000000In1024Slice(b *testing.B) {
+	b.ReportAllocs()
+	chk := New([]*types.FieldType{{Tp: mysql.TypeLong}}, 1024, 1024)
```

Comment:
We can extract a common function for benchmarks, avoid duplicated code.

Reply:
I found `testing.B.Run` is powerful